Library/Formula/cracklib.rb

```diff
@@ -22,5 +25,17 @@ def install
                           "--without-python",
                           "--with-default-dict=#{HOMEBREW_PREFIX}/share/cracklib-words"
     system "make", "install"
+
+    resource("cracklib-words").stage do |r|
+      share.install "cracklib-words-#{r.version}" => "cracklib-words"
+    end
+  end
+
+  def post_install
+    system "#{bin}/cracklib-packer < #{share}/cracklib-words"
```

Comment:
It actually probably doesn't need to be any more, although it is the sort of thing we don't handle directly during install; if nobody else really wants to change it I don't mind where it is now.